frontend/wallet/schema.graphql

```diff
@@ -1,3 +1,6 @@
+scalar UInt64
+scalar PublicKey
```

Comment:
Can you put docstrings on scalars? Might be useful to describe encoding there.

Reply:
there's docstrings on the actual scalars, this schema is pretty much just for the faker